hashintel/hash
# H-1544, H-1592, H-1748: Open draft entities in the slideover instead of the entity editor

## 🌟 What is the purpose of this PR?




This PR opens the entity editor in a slide-over when a draft entity is clicked in the `/drafts` page, replacing the existing inline collapsible properties table.

As a drive-by, this PR also:

- fixes an issue in the entity editor where it displays the button for upgrading an entity's type in readonly mode, resulting in a crash when the button is clicked. This was also an issue in the type editor. (H-1592)
- sets the notification and draft queries which are polled at intervals to use `"network-only"` mode, so that previous query results aren't persisted in memory
- fixes the draft entity type filter to only display the latest version of the types in use on the `/drafts` page
- filter external types from blockprotocol.org from the entities section in the sidebar, except for the `Address` entity type.
- change default sort order on the `/drafts` page to be newest first (H-1748)
- move expensive queries out of the global notifications and draft entities contexts

## 🔗 Related links





- H-1544

## 🔍 What does this change?




See description/commits.

## Pre-Merge Checklist 🚀

### 🚢 Has this modified a publishable library?




This PR:


### 📜 Does this require a change to the docs?




The changes in this PR:


### 🕸️ Does this require a change to the Turbo Graph?




The changes in this PR:


## ⚠️ Known issues



## 🐾 Next steps



## 🛡 What tests cover this?



Manual testing.

## ❓ How to test this?



1. Click on a draft entity in the `/drafts` page, which should open the entity editor in a slide-over
2. Try editing the entity, and click "Save changes" to update the entity

## 📹 Demo




https://github.com/hashintel/hash/assets/42802102/f2f40398-915d-41b7-bfb7-a4529eb4fd8d
